Esteban-Salas-Posada/tarea-4/app.js

```diff
@@ -0,0 +1,80 @@
+// Números
+const zero = document.querySelector('#zero')
+const one = document.querySelector('#one')
+const two = document.querySelector('#two')
+const three = document.querySelector('#three')
+const four = document.querySelector('#four')
+const five = document.querySelector('#five')
+const six = document.querySelector('#six')
+const seven = document.querySelector('#seven')
+const eight = document.querySelector('#eight')
+const nine = document.querySelector('#nine')
+// Operaciones
+const erase = document.querySelector('#erase')
+const percentage = document.querySelector('#percentage')
+const dot = document.querySelector('#dot')
+const plus = document.querySelector('#plus')
+const minus = document.querySelector('#minus')
+const multiply = document.querySelector('#multiply')
+const div = document.querySelector('#div')
+const equal = document.querySelector('#equal')
+const changeSign = document.querySelector('#changeSign')
+// Tablero
+const board = document.querySelector('#board')
+zero.addEventListener('click', function(event){
```

Comment:
@esteban-salas , find a way to assing this listeners with a single function, looks like the is a lot of repited code